src/style/style.css

```diff
@@ -9775,3 +9775,34 @@ ul {
   margin: 0 auto;
   margin-top: 10px;
 }
+.carousel-inner .category-item-title {
+  min-height: 0px;
+  font-size: 1.4rem;
+ }
+ .carousel .left.carousel-control +.right.carousel-control {
```

Comment:
Can you please update the arrow button to remove the gradient and add a width?
ex.
```
.carousel .left.carousel-control +.right.carousel-control {
    width: 65px;
    border-bottom-left-radius: 20px;
    border-top-left-radius: 20px;
    height: 33%;
    top: 33%;
    background-color: gray;
}
```